src/RoadRegistry.Wms.Schema/RoadSegmentDenorm/RoadSegmentDenormRecord.cs

```diff
@@ -0,0 +1,14 @@
+namespace RoadRegistry.Wms.Schema.RoadSegmentDenorm
+{
+    using System;
+
+    public class RoadSegmentDenormRecord
+    {
+        public int Id { get; set; }
```

Comment:
Names need to match verbatim with what is in the old table. This could be tucked away in mapping - allowing the code to deal with English. Could negotiate with the consumer if changes are allowed here.

Reply:
In the other registry which uses WMS, they've placed a SQL view on top which renames the columns to what they need, fyi

Reply:
I'm planning to separate in mapping, yes. WIP though, still just trying some stuff out.

Reply:
A view might indeed be nice - keeps the dutch names to a single location.